python/cudf/cudf/_lib/io/utils.pyx

```diff
@@ -2,17 +2,20 @@
 
 from cpython.buffer cimport PyBUF_READ
 from cpython.memoryview cimport PyMemoryView_FromMemory
+from libcpp.map cimport map
 from libcpp.memory cimport unique_ptr
 from libcpp.vector cimport vector
 from libcpp.pair cimport pair
 from libcpp.string cimport string
 from cudf._lib.cpp.io.types cimport source_info, io_type, host_buffer
-from cudf._lib.cpp.io.types cimport sink_info, data_sink
+from cudf._lib.cpp.io.types cimport sink_info, data_sink, datasource
+from cudf._lib.io.datasource cimport Datasource
 
 import codecs
 import errno
 import io
 import os
+import cudf
 
 # Converts the Python source input to libcudf++ IO source_info
 # with the appropriate type and source values
```

Comment:
use `    if not src:`
https://docs.python.org/3/library/stdtypes.html#truth-value-testing

Reply:
Not sure where in the code you are talking about making this change?

Reply:
https://github.com/rapidsai/cudf/blob/c91cbd478a3946b57f7aa6b2a386278f048a2b4c/python/cudf/cudf/_lib/io/utils.pyx#L23

Reply:
ok done